### DISCUSSION TITLE
国内云安装mdserver-web，简直噩耗

### DISCUSSION THREAD

**Discussion by wuquge** [midoks/mdserver-web#414] [General]
安装实在是慢的跟狗一样，加上时不时的还会出现 安装失败，然后重头再来。直接麻了

![123123213.jpg](https://img1.imgtp.com/2023/05/30/lhWFRkF0.jpg)
![photo_2023-05-29_21-31-53.jpg](https://img1.imgtp.com/2023/05/30/yvShpAoA.jpg)

**Comment by midoks** [comment]
加入国内源。
